src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+
+    {(!age)
+      ? ''
+      : <p className="Person__age">{`I am ${age}`}</p>
+    }
+
+    <p className="Person__partner">
+      { isMarried
+        ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
+        : 'I am not married'
+      }
```

Comment:
Let's move this to a separate variable `${sex === 'f' ? 'husband' : 'wife'}`

Reply:
thank you, I fixed the code according to your sugestions